pkg/policy/trafficdirection.go

```diff
@@ -0,0 +1,46 @@
+// Copyright 2018 Authors of Cilium
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package policy
+
+// TrafficDirection specifies the directionality of policy (ingress or egress).
+// Note that this is deliberately distinct from policymap.TrafficDirection!
```

Comment:
Why is this deliberately distinct from policymap.TrafficDirection? Wouldn't it make more sense to put the type into its own package so it's more portable?